packages/typespec-test/test/multiclient_modular/generated/typespec-ts/src/api/bar/BarContext.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ClientOptions } from "../../common/interfaces.js";
+import { Client } from "../../rest/bar/index.js";
+import { createClient as getClient } from "../../rest/bar/index.js";
+
+export { Client } from "../../rest/bar/index.js";
+
+/** Bar */
+export function createBar(
+  endpoint: string,
+  apiVersion: string,
+  options: ClientOptions = {}
+): Client.BarContext {
+  const baseUrl = endpoint;
+  const clientContext = getClient(baseUrl, options);
```

Comment:
In this case, the api version has a default value 1.0.0 and we put it into options in RLC client factory, should we also put it in the options as give it a default value if it's not set ? 
Another question is what if there're other client level parameters which doesn't exist in the core-client-rest ServiceOptions, does this mean we need common folder even after we migrate some of the types into core-client-rest ?

Reply:
As rest layer will handle this kind of parameter default value, I think it would be better if we keep this clean and filter out any parameter that has a default value in the operation signature.